### ISSUE TITLE
RPA.Browser library is lacking support for multiple browser instances

### ISSUE THREAD

**Issue by mikahanninen** [robocorp/rpaframework#33]
There is need to provide `alias` parameter to `Open Available Browser` keyword. 

User comments about the problem https://robocorp-developers.slack.com/archives/CH1GSKJ3U/p1596380462275000 onwards

workaround for the problem was to create custom keyword
```
*** Keywords ***
My RPA Browser
    [Arguments]   ${alias} 
    ${chrome_options}=  Evaluate  sys.modules['selenium.webdriver'].ChromeOptions()  sys, selenium.webdriver
    Call Method    ${chrome_options}    add_argument    --disable-extensions
    Call Method    ${chrome_options}    add_argument    --headless
    Call Method    ${chrome_options}    add_argument    --disable-gpu
    Call Method    ${chrome_options}    add_argument    --no-sandbox
    Create Webdriver    Chrome    chrome_options=${chrome_options}  alias=${alias}
```

**Comment by seifip** [comment]
Using the My RPA Browser keyword caused errors in the cloud that do not happen when running in a local worker, or using Open Available Browser.

Using OAB does not cause the same errors, but also does not work with Switch Browser.

**Comment by osrjv** [comment]
Support for aliases and regression with `Switch Browser` fixed in release `2.4.0`.
